src/engine/srv.c

```diff
@@ -1261,18 +1257,24 @@ dss_sys_db_init()
 		return rc;
 	}
 
-	if (lmm_db_path == NULL) {
-		D_ERROR("DAOS_LMM_DB_PATH need be configured\n");
+	if (dss_nvme_conf == NULL) {
+		D_ERROR("nvme conf path not set\n");
 		return -DER_INVAL;
 	}
 
+	D_STRNDUP(lmm_db_path, dirname((char *)dss_nvme_conf), PATH_MAX);
```

Comment:
I guess 'dss_nvme_conf' is truncated by this dirname() call, that caused failures.

Reply:
I had the string copy before https://github.com/daos-stack/daos/pull/11319/commits/80ee66a77260dfcf23bccc2b3094a78532c15a79#diff-edb42428ecfead9743e3ea5d226e755fb73cb56ec4e90158bd4f5f6ded669137L1270 but was asked to remove it, I will reinstate it

Reply:
https://github.com/daos-stack/daos/pull/11319#discussion_r1092018870